packages/orchestrator-ui-components/src/rtk/endpoints/subscriptionInUseByRelationsList.ts

```diff
@@ -0,0 +1,54 @@
+import { CacheTags, orchestratorApi } from '@/rtk';
+import { InUseByRelationDetail, InUseByRelationsDetailResult } from '@/types';
+
+export const subscriptionInUseByRelationQuery = `
+    query SubscriptionsInUseByRelationsDetails(
+        $subscriptionIds: String!
+    ) {
+        subscriptions(
+            first: 1000000
```

Comment:
I don't think this is the correct way to get all in use by subscriptions. It's paginated, so probably easiest is to write some function that iterates over all pages and yields the results.

Reply:
We have had this discussion before and I don't think it's the right way to get all results by doing extra queries just to make sure there aren't any extra results. That means extra unneeded http requests.

Reply:
You can have both. A collector with a default fetch size of for example 1000 (or any other reasonable number), that gets all your items in batches of 1000. And yes, there will be additional requests if you have 1001 items. And the possibility to override the default batch size if you really want to get data in larger chunks. My point is mainly that you want to abstract away from pages in these cases, and avoid having magic numbers all over the place.

Reply:
Replaced the number with the const. To be complete. We have in fact implemented paging as you described for pages that actually have paging. For pages that just need everything the results of discussions with backenders have been to not implement something for that so all we have at the frontend is using a large number or making unneeded requests just to satisfy some code esthetics. My choice here is to make what we do explicit and not do the unneeded calls